webview/src/plots/components/comparisonTable/ComparisonTableRow.tsx

```diff
@@ -129,56 +133,56 @@ export const ComparisonTableRow: React.FC<ComparisonTableRowProps> = ({
         id={path}
         ref={bodyRef}
       >
-        <tr>
-          <td
-            className={cx({ [styles.pinnedColumnCell]: pinnedColumn })}
-            colSpan={pinnedColumn ? 1 : nbColumns}
+        <ComparisonTablePinnedContentRow
```

Comment:
I decided to keep the path sticky but we also could keep it static as before: 

https://github.com/iterative/vscode-dvc/assets/43496356/7d704cc4-d27a-45b4-a86d-90401d9d258a

Reply:
I feel like it was better before. It feels weird to move the classes but not the path.